assets/wizards/engagement/views/social/index.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Social screen.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { ExternalLink } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { ActionCard, Card, Button, withWizardScreen } from '../../../../components/src';
+
+/**
+ * Social Screen
+ */
+class Social extends Component {
+	/**
+	 * Render.
+	 */
+	render() {
+		const { connected, connectURL } = this.props;
+
+		return (
+			<ActionCard
+				title={ __( 'Jetpack Publicize' ) }
```

Comment:
Publicize and Social Icons are both on the Jetpack settings page this links. We should probably also call attention to the Social Icons feature here. Sites can customize the style of the buttons and whether they want them on posts or not. Not a blocker for this PR, though; we'll address when we fully build out this section.